rrfs-test/ush/wrapper_ctest_pbs.sh

```diff
@@ -31,7 +31,7 @@ ulimit -v unlimited
 ulimit -a
 export OOPS_TRACE=0
 export OMP_NUM_THREADS=1
-export LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:${RDASApp}/build/lib"
+export LD_LIBRARY_PATH="${RDASApp}/build/lib64:${RDASApp}/build/lib:${LD_LIBRARY_PATH}"
```

Comment:
`lib64` is a link to `lib`,
I think, similar to what we did in rrfs-workflow, using `lib64` only (ie. without `lib`) may be enough.

https://github.com/NOAA-EMC/rrfs-workflow/blob/rrfs-mpas-jedi/workflow/sideload/launch.sh#L69